src/VisualStudio/Setup/Roslyn.VisualStudio.Setup.csproj

```diff
@@ -349,6 +349,7 @@
     <None Include="source.extension.vsixmanifest">
       <SubType>Designer</SubType>
     </None>
+    <Content Include="UnifiedSettings\*.registration.json" IncludeInVSIX="true" />
```

Comment:
💡 Should be placed in the item group above that contains other `<Content>` items